code/modules/clothing/outfits/ert.dm

```diff
@@ -62,6 +64,7 @@
 	glasses = /obj/item/clothing/glasses/hud/security/sunglasses
 	back = /obj/item/storage/backpack/security
 	belt = /obj/item/storage/belt/security/full
+	r_hand = /obj/item/storage/briefcase/lasergun
```

Comment:
Default Security doesn't need gangster style gun cases.

Reply:
Its to stop bag null space